docs/docusaurus/docs/get_support.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_label: 'Get support'
+title: 'Get support'
+description: Learn what resources are avialable to help you resolve issues with GX Core.
+---
+
+GX Core is open source software (OSS) and relies on an active user community for its continued success. Input from users like you help GX and the community build its collective knowledge and improve GX Core. Knowing how and where to seek support will help you get the answers you need to resolve issues quickly.
+
+## Try solving your issue first
+
+The following are some recommendations for resolving issues on your own. Resolving issues on your own builds your own knowledge and provides you with an opportunity to share that knowledge with the community. 
+
+### Search the GX documentation
+
+Search the docs you're using currently for an answer to your issue or question. If you're new to GX Core, try the [Quickstart](tutorials/quickstart/quickstart.md) or one of the specific getting started guides such as [Get started with Great Expectations and Databricks](tutorials/getting_started/how_to_use_great_expectations_in_databricks.md), or [Get started with Great Expectations and SQL](tutorials/getting_started/how_to_use_great_expectations_with_sql.md).  
+
+### Search with your favorite browser
+
+It's very likely that someone, somewhere has previously encountered or solved the issue you're currently experiencing. Perform a keyword search in your favorite browser to discover resources outside GX documentation. 
+
+### Try it out
+
+If you're wondering what might happen when you implement a solution, try it out! If you attempt a solution in your development environment, users are unaffected and unaware of what you're up to. Sneaky. 
+
+## Ask the right questions and provide the right information
+
+Taking the time to formulate your questions properly and providing all the relevant background information can save time and improve response times. The following are some recommendations that will help the GX Developer Advocates (DAs) help you. 
+
+### Include all the relevant information
+
+Tell the GX DAs exactly what issue you're experiencing, or the problem you're trying to solve. When asking questions, you should provide the following:
```

Comment:
I'd recommend: "Clearly explain the issue you're experiencing, ...." since it's not always going to be DAs who respond.

Reply:
@rachhouse the edit was implemented as requested.